crates/rust-analyzer/src/dispatch.rs

```diff
@@ -10,14 +10,30 @@ use crate::{
     LspError, Result,
 };
 
+/// A visitor for routing a raw JSON request to an appropriate handler function.
+///
+/// Most requests are read-only and async and are handled on the threadpool
+/// (`on` method).
+///
+/// Some read-only requests are latency sensitive, and are immediately handled
+/// on the main loop thread (`on_sync`). These are typically typing-related
```

Comment:
@Veykril ``[`RequestDispatcher::on_sync`]`` doesn't work for me. Is it our bug, or is this just isn't supported by rustdoc?

Reply:
Should work I think.

Reply:
Ye I'm pretty sure that should work with rustdoc. Odd that we don't resolve this 🤔

Reply:
Oh right this is https://github.com/rust-analyzer/rust-analyzer/issues/9694